Library/Formula/hidapi.rb

```diff
@@ -4,6 +4,20 @@ class Hidapi < Formula
   homepage 'https://github.com/signal11/hidapi'
   url 'https://github.com/signal11/hidapi/archive/hidapi-0.8.0-rc1.tar.gz'
   sha1 '5e72a4c7add8b85c8abcdd360ab8b1e1421da468'
+  
+  ## Patch - April 8th, 2015 - Raphaël P. Barazzutti
```

Comment:
Thanks! You're welcome to get a second opinion from the maintainers, but we don't have many places where patch authors are written into formulae :).